component-samples/demo/scripts/web_csr_req.sh

```diff
@@ -22,3 +22,5 @@ openssl x509 -x509toreq -in webServer.pem -out server.req -signkey server-key.pe
 #comment out following line if signing with external CA
 openssl x509 -req -days 12775 -in server.req -CA ca-cert.pem -CAkey caKey.pem -CAcreateserial -out server-cert.pem -extfile ./web-server.conf -extensions v3_req  
 
+chmod 644 server-key.pem
```

Comment:
We generally don't run chmod in a script.
https://stackoverflow.com/questions/68063830/including-chmod-in-shell-script-will-it-work-for-everyone
Least Privilege Principle.

We ask customers to `chmod -R 644 secrets/ service.env`